fixtures/git_content_repos.py

```diff
@@ -1,10 +1,36 @@
 import pytest
+import json
+import urllib
+import urllib.error
+import urllib.parse
+import urllib.request
+
+import requests
 
 
 @pytest.fixture
-def git_content_repos():
-    """Returns the list of all github content repos
+def git_content_repos(headers_data):
+
+    """Returns the list of all github content repos (starting with osbooks-)
     """
-    repo = ["osbooks-university-physics", "osbooks-college-algebra-bundle"]
 
-    return repo
+    url = "https://api.github.com/orgs/openstax/repos?per_page=50&type=private"
+
+    resp = requests.get(url, headers=headers_data)
+
+    repos = []
+
+    while resp.links.get("next"):
+        next_url = resp.links["next"]["url"]
+        resp = requests.get(next_url, headers=headers_data)
+
+        repos_req = urllib.request.Request(resp.url, headers=headers_data)
+        repos_list = urllib.request.urlopen(repos_req).read()
```

Comment:
This is minor, but you end up requesting the same data here as you do in L25 via `requests`. I suspect this may be because you're using `requests` to get the paging links in the header since it parses those automatically for you. However, if you want to reduce this network overhead, you can either:

1. Parse the same headers with urllib manually (e.g. they're available in the response `urllib.request.urlopen(repos_req).headers.get('Link')`). 
2. Remove use of `urllib` altogether in this fixture and drop L27-28 by making L31 `for item in resp.json():`.